bob-access/src/authenticator/users_storage/config.rs

```diff
@@ -0,0 +1,76 @@
+use std::collections::HashMap;
+
+use crate::error::Error;
+
+use super::{Perms, User};
+
+#[derive(Debug, Clone, Deserialize)]
+pub(super) struct ConfigUser {
+    pub(super) username: String,
+    pub(super) password: String,
+    pub(super) role: String,
```

Comment:
Don't you think user should be allowed to have multiple roles? We can remove `Admin` role in example then, and just leave `reader` and `writer`

Reply:
We implement a claim-based policy. Role in this case is just a group of claims, so we don't need to support multiple roles. What we need is the ability to specify claims directly for each user. In the first version we can keep the current logic, and I will create a new issue to add this ability in the future

Reply:
https://github.com/qoollo/bob/issues/408